go.mod

```diff
@@ -1,30 +1,32 @@
 module decred.org/dcrctl
 
-go 1.22
+go 1.23
+
+toolchain go1.23.6
```

Comment:
i thought you hated this

Reply:
Meh, I missed that it added it.  I'm not sure why it did.  Removed.  Thanks.